tools/loader/Loader.cpp

```diff
@@ -341,20 +341,25 @@ void Loader::compile(PlaceholderBindings &bindings) {
 
   CompilationContext cctx;
   cctx.mode = CompilationMode::Infer;
-  if (emittingBundle()) {
-    // Emit IR for the graph, compile it and save as a bundle.
-    EE_.save(F_, cctx, emitBundle, networkName);
-  } else {
-    // Emit IR for the graph and compile it.
-    EE_.compile(F_, cctx);
-  }
 
   if (dumpGraphOpt) {
     F_->dump();
   }
   if (!dumpGraphDAGFileOpt.empty()) {
     F_->dumpDAG(dumpGraphDAGFileOpt.c_str());
```

Comment:
This is now dumping the unoptimized graph instead of the optimized one. Since the Function should still be valid now, can you move this back below the calls to `addNetwork()`/`save()` below?

Reply:
I moved them back below addNetwork()/save() and now walk all functions in the module to dump them. This handles the case of partitioning.